cards/document-standard/template.hbs

```diff
@@ -76,10 +76,10 @@
     {{#if ariaLabel}}aria-label="{{ariaLabel}}"{{/if}}>
     {{#if (any iconName iconUrl)}}
     <div class="HitchhikerCTA-iconWrapper">
-      <div class="HitchhikerCTA-icon" data-component="IconComponent" data-opts='{
-        "iconUrl": "{{#if iconUrl}}{{#unless (isNonRelativeUrl iconUrl)}}{{@root.relativePath}}/{{/unless}}{{iconUrl}}{{/if}}",
-        "iconName": "{{iconName}}"
-      }'></div>
+      {{> icons/iconPartial
+        iconName=iconName
```

Comment:
do you know why an extra space is showing up on these diffs? It might not be an issue

Reply:
not sure I see the extra spaces